docs/conf/eu/2016/news/thanks-for-coming.md

```diff
@@ -0,0 +1,64 @@
+
+We hope you had a wonderful time, and that you are as excited as we are about the beautiful community that is forming. Below we've included all of the resources you might need for staying in touch, like the Write the Docs Slack. We also have a short anonymous survey for you to fill out, so we can make the conference even better next year.
+
+## Tell us how we did
+
+Let us know how we can improve for future conferences and gatherings. We really appreciate your candid feedback!
+
+**Our three question survey is quick and easy, so [go fill it out](https://goo.gl/forms/RQE7qyaFvaMNP2Zo2).**
+
+## Speak, Sponsor, or Organize next year
+
+This is the end of our 2016 round of conferences,
+but that doesn't mean things are over!
+We'll be having another North American and European conference next year.
+
+We're always looking for folks who want help us organize the events.
+We have worked to document the [roles](http://www.writethedocs.org/organizer-guide/confs/event-roles/) that need to happen to put the conference together.
+We could use a few folks who are interested in lending a hand to make the event happen,
+so please let us know if that's you!
+
+We also depend on sponsors to make our events as magical as they are,
+while keeping ticket prices low.
+If you are curious about sponsoring Europe or North America next year,
+please contact us at [sponsorship@writethedocs.org](mailto:sponsorship@writethedocs.org).
+
+Also don't forget that inspiration and start thinking of your topic ideas for next year.
+We'll be opening our North American CFP in December,
+and Europe will be open in April next year.
+
+## Photos and Videos
+
+The [official photos](https://www.flickr.com/photos/writethedocs/sets/72157674031974295) from the conference are now available.
+Thanks to [Kay Smoljak](https://twitter.com/goatlady) for putting them together so quickly.
+They are licensed for non-commercial use with attribution,
+so feel free to use them in write-ups about the event.
+
+All the videos for the conference will be posted soon.
+We'll send another update once they're available!
+
+## Write the Docs Slack
+
+The Write the Docs Slack has really taken off this year! Our general Slack channel has been extra busy in the days and weeks leading up to the conference, and we hope that y'all continue the conversations throughout the year. If you haven't joined, head on over and join us! We're at [Write the Docs Slack](http://slack.writethedocs.com/).
+
+## Meetups
+
+We hope you continue to [build the community](http://www.writethedocs.org/meetups/) and keep in touch with other documentarians in your hometown. In this spirit, we have a number of meetups in cities across the country. If you don't see your city [listed](http://www.writethedocs.org/meetups/), consider [starting](http://www.writethedocs.org/meetups/starting/) a meetup! 
```

Comment:
"across the country" sounds like leftovers from NA, maybe we can rephrase for global?